csv2bufr/cli.py

```diff
@@ -110,12 +116,38 @@ def transform(ctx, csv_file, mapping, output_dir, station_metadata, verbosity):
         except Exception as err:
             raise click.ClickException(err)
 
+    # load JSON json_template
+    if json_template is not None:
+        if not os.path.isfile(json_template):
+            json_template_file = f"{MAPPINGS}{os.sep}{json_template}.json"
+        else:
+            json_template_file = json_template
+        try:
+            with open(json_template_file) as fh:
+                json_template = json.load(fh)
+        except Exception as err:
+            raise click.ClickException(err)
+
     click.echo("Writing data to file")
     for item in result:
         filename = f"{output_dir}{os.sep}{item}.bufr4"
         with open(filename, "wb") as fh:
             fh.write(result[item].read())
 
+        # convert to JSON if template specified
+        if json_template is not None:
```

Comment:
We should move the JSON serialization such that the `csv2bufr` module returns a dict of GeoJSON and then the CLI serializes it.  This will allow other downstream applications to receive JSON data without going through the CLI per se.